src/components/PauseLiquidations.tsx

```diff
@@ -7,11 +7,15 @@ interface Props {
   charterOfferId: number;
 }
 
+const invitationDescriptions = {
+  newBid: 'new bid',
```

Comment:
can this be imported from agoric-sdk?

Reply:
It's not exported at the moment: https://github.com/Agoric/agoric-sdk/blob/ec6a7f4eded916afb8c927638b4a699e2ffd99c9/packages/inter-protocol/src/auction/auctioneer.js#L564

Reply:
@Chris-Hibbert @dckc WDYT of putting exporting invitation description string makers from the contracts and re-exporting them through clientSupport.js so clients can produce the strings they need to look up in vstorage?

Reply:
exporting the description constants via `clientSupport.js` makes sense.

I'm not sure what you mean by "string makers".

Reply:
We don't do this for any of the other "pause" features. Probably worth cataloging and doing them all at once if it's worth doing.

Reply:
> exporting the description constants via `clientSupport.js` makes sense.
> 
> I'm not sure what you mean by "string makers".

They're not all constants. E.g. https://github.com/Agoric/agoric-sdk/blob/841a7012831166f6c29f00bdffc907f2b35a7b0c/packages/inter-protocol/src/vaultFactory/vaultManager.js#L321-L323

> We don't do this for any of the other "pause" features. Probably worth cataloging and doing them all at once if it's worth doing.

agreed, not a blocker for this PR or Vaults Release https://github.com/Agoric/agoric-sdk/issues/7312

Reply:
> They're not all constants.

That seems like a bug. IIUC, folks are supposed to be able to statically find the description that stands for the role they're going to play. Anything dynamic belongs in extraProperties (sp?).